File: 645-set-mismatch/set-mismatch.js

var findErrorNums = function(nums) {
    let dup = -1, missing = -1;
    
    for (let i = 1; i <= nums.length; i++) {
        let count = nums.filter(num => num === i).length;
        if (count === 2) {
            dup = i;
        } else if (count === 0) {
            missing = i;
        }
    }
    
    return [dup, missing];
};

File: 645-set-mismatch/set-mismatch.py
class Solution:
    def findErrorNums(self, nums):
        dup, missing = -1, -1
        
        for i in range(1, len(nums) + 1):
            count = nums.count(i)
            if count == 2:
                dup = i
            elif count == 0:
                missing = i
        
        return [dup, missing]

